src/manage_nsfs/manage_nsfs_events_utils.js

```diff
@@ -384,4 +384,30 @@ NoobaaEvent.CONFIG_DIR_UPGRADE_FAILED = Object.freeze({
     state: 'DEGRADED',
 });
 
+///////////////////////////////
+//   NOTIFICATION EVENTS     //
+///////////////////////////////
+
+NoobaaEvent.NOTIFICATION_LOW_SPACE = Object.freeze({
+    event_code: 'noobaa_notification_low_space',
+    message: 'Pending notification log dir low on space',
+    description: 'Low space',
+    entity_type: 'NODE',
+    event_type: 'WARN',
+    scope: 'NODE',
+    severity: 'WARN',
+    state: 'HEALTHY',
+});
+
+NoobaaEvent.NOTIFICATION_FAILED = Object.freeze({
```

Comment:
would this be relevant when other notifications will fail as well?

Reply:
I'm not sure I understand.
You mean notifications other than bucket event notifications?
If so, maybe? It's possible, other places would use it for different purposes?
I can change the name to BUCKET_NOTIFICATION_FAILED if that would help.

Reply:
you can change it to BUCKET_NOTIFICATION_FAILED if you think it needs to be used only in this context. but maybe we should send this notification in other places. we can also do it in the future on a different PR